aries_cloudagent/protocols/present_proof/v2_0/messages/pres_format.py

```diff
@@ -52,6 +56,15 @@ def aries(self) -> str:
             """Accessor for aries identifier."""
             return self.value.aries
 
+        @property
+        def handler(self) -> Type["V20PresFormatHandler"]:
```

Comment:
The handlers are not defined yet.

See https://github.com/animo/aries-cloudagent-python/blob/cwu-bbs-main/aries_cloudagent/protocols/issue_credential/v2_0/messages/cred_format.py#L40-L48